Code/MinimalLib/jswrapper.cpp

```diff
@@ -372,6 +372,15 @@ emscripten::val get_avalon_fp_as_uint8array(const JSMol &self) {
 }
 #endif
 
+#ifdef RDK_BUILD_MINIMAL_LIB_FRAGMENTATION
+emscripten::val get_matched_fragments(const JSMol &mol, unsigned int minCuts, unsigned int maxCuts, unsigned int maxCutBonds) {
```

Comment:
I can create a string with parameters and parse it but as get_matches has up to 3 explicit options I considire this ok